homeassistant/components/myq/cover.py

```diff
@@ -104,37 +101,48 @@ def unique_id(self):
 
     async def async_close_cover(self, **kwargs):
         """Issue close command to cover."""
-        self._last_action_timestamp = time.time()
-        await self._device.close()
-        self._async_schedule_update_for_transition()
+        if self.is_closing or self.is_closed:
```

Comment:
This is similar to `async_open_cover`.  Maybe it could be a bit more DRY

Reply:
Actually there are so many different properties being checked here, its not going to save that much.